igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/GitlabCiConfig.java

```diff
@@ -40,13 +41,13 @@
 import retrofit.converter.JacksonConverter;
 
 @Configuration
-@ConditionalOnProperty("gitlab-ci.enabled")
+@ConditionalOnProperty("gitlabci.enabled")
```

Comment:
This will break folks that use the old property...can we stick with the old one to prevent that?

Reply:
Doesn't Spring convert hyphenated and camel case natively? i.e. gitlabCi === gitlab-ci. Not sure what the rational is for changing it though.

Reply:
Reverted to the old one and re-tested, it does work.  I think this was just a mistake on my end, learning Spring while doing this PR and I thought here were some places that were `gitlabci` and some `gitlab-ci` but looking back I don't see them on master.  I think I may have confused them with some of the `gitlab` (no CI) config?  Not sure, but either way reverted it